landlab/components/flow_accum/flow_accumulator.py

```diff
@@ -548,6 +550,25 @@ class DepressionFinderAndRouter to the parameter `depression_finder`.
     ...                      depression_finder=DepressionFinderAndRouter,
     ...                      routing='D4')
 
+    FlowAccumulator was designed to work with all types of grids. However,
+    NetworkModelGrid's have no cell area. Thus, in order for FlowAccumulator to
+    this type of grid, an at-node array called ``cell_area_at_node`` must be
+    present.
+
+    >>> from landlab.grid.network import NetworkModelGrid
+    >>> y_of_node = (0, 1, 2, 2)
+    >>> x_of_node = (0, 0, -1, 1)
+    >>> nodes_at_link = ((1, 0), (2, 1), (3, 1))
+    >>> nmg = NetworkModelGrid((y_of_node, x_of_node), nodes_at_link)
+    >>> area = nmg.add_ones('node', 'cell_area_at_node')
+    >>> z = nmg.add_field('topographic__elevation',
```

Comment:
Why not use black formatting in docstrings too?
```python
    >>> z = nmg.add_field(
    ...     'topographic__elevation',
    ...     nmg.x_of_node + nmg.y_of_node,
    ...     at = 'node',
    ... )
```

Reply:
Can do. One thing to note, however, as we've been starting to be much better about lint... Is that ([to the best of my knowledge](https://github.com/ambv/black/issues/144)) black does not yet support formatting docstrings... @mcflugen correct me if you know better.

But there is something called [docformatter](https://github.com/myint/docformatter) which does. I've tried it out with mixed success.

Reply:
My attempt at the middle path resulted in changing the formatting for FlowAccumulator, and all FlowDirectors... Definitely looks nicer!